five.js

```diff
@@ -85,7 +85,8 @@
   five.thai = function() { return 'ห้า'; };
   five.ukrainian = function() { return 'п’ять'; };
   five.welsh = function() { return 'pump'; };
-
+  five.sinhala = function() { return 'පහ'; };
```

Comment:
Would you mind moving this into the correct alphabetical place please? Same for test.js :D